### DISCUSSION TITLE
Running addGeneExpressionMatrix() meets an error

### DISCUSSION THREAD

**Discussion by Jiahuan90** [GreenleafLab/ArchR#1451] [Questions / Documentation] [answered]
Describe the question:

I ran demo data from getTutorialData("Hematopoiesis") with :
>addArchRGenome("hg19")

>ArrowFiles <- createArrowFiles(
  inputFiles = inputFiles,
  sampleNames = names(inputFiles),
  minTSS = 4, #Dont set this too high because you can always increase later
  minFrags = 1000,
  addTileMat = T,
  addGeneScoreMat = T
)

And met the error that
1. 2022-06-04 16:16:50 : ERROR Found in .addGeneScoreMat TmpGS for scATAC_PBMC_R1
<simpleError in x[, i, drop = FALSE] - y[, j, drop = FALSE]: non-numeric argument to binary operator>

2. 2022-06-04 16:17:01 : ERROR Found in .addGeneScoreMat for (scATAC_PBMC_R1 : 3 of 3)
<simpleError in .safelapply(seq_along(geneRegions), function(z) {    totalGSz <- tryCatch({.......

I followed the full manual in the webpage and already searched the github.

I don't know what's going wrong?

Thank you for your kindly help~~
Below is the log file:

[ArchR-createArrows-12dac5d789f27-Date-2022-06-04_Time-15-54-07.log](https://github.com/GreenleafLab/ArchR/files/8837442/ArchR-createArrows-12dac5d789f27-Date-2022-06-04_Time-15-54-07.log)

**Comment by rcorces** [comment, marked as answer]
I apologize for the delay in my reply. I'm not sure what is causing your error. Typically, if you are encountering issues with the tutorial code and tutorial data, it is an issue with your environment and those are very challenging to troubleshoot.

**Reply by Jiahuan90** [reply]
Hi rcorces, I have solved the problem. The error is that I load too many R package at same time, and different R packages may incompetiable with each other and met unknown error. When I only load ArchR, it was successful! I think we could close the topic. Thank you for your help!
